### DISCUSSION TITLE
can't get the timer on screen to go off

### DISCUSSION THREAD

**Discussion by cloes478** [SelfControlApp/selfcontrol#687] [Q&A]
I have started using the Self Control app and I really like it. Now, the timetracker on the side of my internet taps is always there keeping track of "time spent on ___" won't go off after I quit the app and x out tabs. CAn someone please help me take this off without deleting the app? It's very frustrating and distracting. 
Thank you in advance !

**Comment by cstigler** [comment]
Hi @cloes478, hopefully you've figured this out by now, but the "Time spent on ___" time tracker tab isn't SelfControl! SelfControl doesn't add any tabs to your web browser.

It sounds like you might have added another distraction-blocker, maybe an extension in your Chrome/Firefox browser. I'd suggest looking through your extensions and disabling them until you find the culprit.
